runelite-client/src/main/java/net/runelite/client/plugins/grotesqueguardians/GrotesqueGuardiansOverlay.java

```diff
@@ -86,31 +85,28 @@ public Dimension render(Graphics2D graphics)
 			if (graphicsObject.getId() >= GraphicID.GROTESQUE_GUARDIANS_LIGHTNING_BEG && graphicsObject.getId() <= GraphicID.GROTESQUE_GUARDIANS_LIGHTNING_END)
 			{
 				color = Color.ORANGE;
-				text = "Lightning";
 			}
 
 			if (graphicsObject.getId() == GraphicID.GROTESQUE_GUARDIANS_STONE_ORB)
 			{
 				color = Color.GRAY;
-				text = "Stone Orb";
 			}
 
 			if (graphicsObject.getId() == GraphicID.GROTESQUE_GUARDIANS_FALLING_ROCKS)
 			{
 				color = Color.YELLOW;
-				text = "Falling Rocks";
 
 			}
 
 			if (color != null)
 			{
 				OverlayUtil.renderPolygon(graphics, poly, color);
 
-				Point textLocation = Perspective.getCanvasTextLocation(client, graphics, lp, text, 0);
+				Point textLocation = Perspective.getCanvasTextLocation(client, graphics, lp, "", 0);
```

Comment:
Is there any reason to keep the `textLocation` code for an empty string? Remove this and the call to `renderTextLocation`.

Reply:
👍 Thank you. Derped on that one.